### DISCUSSION TITLE
EMAIL_ENCRYPTION_METHOD env variable

### DISCUSSION THREAD

**Discussion by mikewink** [craftcms/cms#9593] [Ideas] [answered]
Hello Craft CMS team,

I wanted to ask if we could have a `EMAIL_ENCRYPTION_METHOD` env variable, just like the other email related ones. I know the select does not allow for setting variable name itself, but based on the convention I think we should at least check if there is on set in .env and use it.
Another option would be to add a fourth select option named `from variable` or just `variable`.

The reason I'm asking is that I push config from local to production and on my local machine I use a non SSL SMTP server connection, while in production the system is connected via SSL. So right now I have to very cautious what I push, so I don't push my local non SSL encryption method settings to production. 

Thank you very much for considering.

**Comment by brandonkelly** [comment, marked as answer]
This is basically the same request as #8866 – which we are going to get into Craft 4 😄

**Reply by mikewink** [reply]
Ohh that's good news Brandon. Thank you very much.
